changelog/unreleased/issue-2571

```diff
@@ -0,0 +1,15 @@
+Bugfix: Check contents of unchanged files
+
+We've added an extra check wheter the contents are available in the index when backing
+up unchanged files.
+
+Backup used to check if file were unchanged and in this case did not process these
+file at all - even if their contents were not available, maybe due to an error that
+was manually fixed. 
```

Comment:
Backup still checks if the file is unchanged. I would change process to save.

I would list disk corruption or accidental removal as the prime causes of the problem.

Reply:
changed.